app/upgrades/v0_3_0/upgrades.go

```diff
@@ -15,6 +16,7 @@ const (
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
+	StoreUpgrades:        storetypes.StoreUpgrades{Added: nil, Renamed: nil, Deleted: nil},
```

Comment:
not sure if I like editing previous migrations vs just adding "ignore lint" comment (here and above and in other old files of migrations)